classes/local/store/object_file_system.php

```diff
@@ -623,10 +623,23 @@ public function copy_file_from_hash_to_path($contenthash, $destinationpath) {
      *
      * @param string $contenthash file to be moved
      */
-    public function delete_external_file_from_hash($contenthash, $force = false) {
+    public function delete_external_file_from_hash($contenthash, $force = false, $force_delay_delete = false) {
```

Comment:
```suggestion
    public function delete_external_file_from_hash($contenthash, $force = false, $forcedelaydelete = false) {
```
Params names in Moodle don't have underscores.

Also, please update the doc block for this method so it has all parameters. Doc blocks help developers understand what the params are for ;-)